blackbox/docs/sql/statements/values.rst

```diff
@@ -0,0 +1,44 @@
+.. highlight:: psql
+.. _ref-values:
+
+==========
+``VALUES``
+==========
+
+``VALUES`` computes a set of rows.
+
+
+Synopsis
+========
+
+::
+
+    VALUES ( expression [, ...] ) [, ...]
+
+
+Description
+===========
+
+``VALUES`` can be used to generate a result set containing constant values.
+
+When more than 1 row is specified, all rows must have the same number of
+elements.
+
+An example::
+
+   cr> VALUES (1, 'one'), (2, 'two'), (3, 'three');
+   +------+-------+
+   | col1 | col2  |
+   +------+-------+
+   |    1 | one   |
+   |    2 | two   |
+   |    3 | three |
+   +------+-------+
+   VALUES 3 rows in set (... sec)
+
+
+It is commonly used in :ref:`ref-insert` to provide values to insert into a
+table.
+
+The expressions across rows that make up the same columns must have the same
```

Comment:
rows and columns need to be swapped here? ie. "across columns that make up the same rows"

Reply:
Hm, I'm not sure, what's meant is that in:

```
   | c1 | c2
r1 | 1  | a
r2 | 2  | b
```

The expressions in c1 must all have the same type, and the same for all expressions within c2. Wasn't sure how to express that better.

Reply:
Hm, I see, it's still very non-descriptive imo. Would  "The expressions executed against each column must have the same type" ?